test/wait_for_blocks.go

```diff
@@ -0,0 +1,74 @@
+package test
+
+import (
+	"context"
+
+	"golang.org/x/sync/errgroup"
+)
+
+// ChainHeighter fetches the current chain block height
+type ChainHeighter interface {
+	Height(ctx context.Context) (uint64, error)
+}
+
+// WaitForBlocks blocks until all chains reach a block height delta equal to or greater than the delta argument
+func WaitForBlocks(ctx context.Context, delta int, chains ...ChainHeighter) error {
```

Comment:
I had to use a different package `test` because of import cycles. May be worth changing in the future. 

This demonstrates how to use a tiny interface `ChainHeighter` in this case. 

For now, I had to keep `Height()` in the `ibc.Chain` interface because the factory returns the iface, not a concrete type.

Reply:
I would have expected this to fit nicely in the root `ibctest` package, but it is used in the chain implementations, and `ibctest` imports those chains. 

Package `test` is fine for now, but I would like to see it renamed once we figure out what else fits in here.